metagraph/tests/algorithms/test_embedding.py

```diff
@@ -58,11 +48,75 @@ def cmp_func(embedding):
                 assert a_to_a_center < a_to_b
                 assert b_to_b_center < a_to_b
 
-    embedding.normalize(dpr.types.NodeEmbedding.NumpyNodeEmbeddingType).custom_compare(
+    MultiVerify(dpr).compute(
+        "embedding.train.node2vec",
+        graph,
+        p,
+        q,
+        walks_per_node,
+        walk_length,
+        embedding_size,
+        epochs,
+        learning_rate,
+    ).normalize(
+        (dpr.types.Matrix.NumpyMatrixType, dpr.types.NodeMap.NumpyNodeMapType)
+    ).custom_compare(
         cmp_func
     )
 
 
+def test_graph2vec(default_plugin_resolver):
+    try:
+        from sklearn.mixture import GaussianMixture
+    except:
```

Comment:
Change this to `except ImportError:`. Bare excepts should be avoided.

Reply:
https://github.com/metagraph-dev/metagraph/pull/138/commits/c6f88dfd8b708da632b8f4336400a510ed49f371 makes it so that it catches `ModuleNotFoundError`.